types/experiments.ts

```diff
@@ -0,0 +1,72 @@
+/** All fields are publically accessible via the API and available to clients */
+export interface Experiment {
+  /** Unique identifier for the experiment */
+  slug: string;
+  /** Publically-accesible name of the experiment */
+  public_name: string;
+  /** Short public description of the experiment */
+  public_description: string;
+  /** Experimenter URL */
+  experiment_url: string;
+
+  /** Is the experiment currently live in production? i.e., published to remote settings? */
+  is_published: boolean;
+  /** Are we continuing to enroll new users into the experiment? */
+  is_enrollment_paused: boolean;
+
+  /** Reference to an Audience, which contains targeting information */
+  audience: Audience;
+  /** Bucketing configuration */
+  bucket_config: BucketConfig;
+
+  /** A list of cusom feature metrics relevant to the experiment analysis */
+  feature_metrics: Array<FeatureMetric>;
```

Comment:
I think @tdsmith has explained to me that this will be a list of features, and the metrics that correspond to those features will live only in Pensieve?

Reply:
:+1: